src/App/App.tsx

```diff
@@ -3,31 +3,24 @@ import './App.scss';
 import { Routes, Route } from 'react-router-dom';
 import Dashboard from '../Components/Dashboard/Dashboard';
 import WritingPage from '../Components/WritingPage/WritingPage';
-import { ApolloClient, InMemoryCache, ApolloProvider, useLazyQuery } from '@apollo/client';
-import { GET_SINGLE_USER } from '../Queries';
 
-const client = new ApolloClient({
-  cache: new InMemoryCache(),
-  uri: `https://one-minute-writer-be.herokuapp.com/graphql`,
-})
+import { GET_SINGLE_USER } from '../Queries';
 
 const App: React.FC = () => {
 
-  const [ fetchUser, { loading, error, data } ] = useLazyQuery(GET_SINGLE_USER, {
-    variables: {
-      id: 1,
-    }
-  })
+  // const [ fetchUser, { loading, error, data } ] = useLazyQuery(GET_SINGLE_USER, {
```

Comment:
Do you really need to keep this commented code? If it's not useful code(dead code), we should remove it. This makes it hard for the future devs to understand. If it's necessary to keep please write a note about why it's nice to keep.

Reply:
I'll delete